src/settings-ui/Settings.UI/Strings/en-us/Resources.resw

```diff
@@ -3646,6 +3657,7 @@ Activate by holding the key for the character you want to add an accent to, then
   </data>
   <data name="GeneralPage_EnableExperimentation.Description" xml:space="preserve">
     <value>Note: Only Windows Insider builds may be selected for experimentation</value>
+	<comment>{Locked="Windows Insiders"}</comment>
```

Comment:
`Windows Insiders` is not in the string. Only `Windows Insider` is.

Locks require perfect string matches.